app/main.py

```diff
@@ -0,0 +1,112 @@
+import asyncio
+import logging
+
+from aiogram import Bot, Dispatcher, Router, types
+from aiogram.filters import Command
+from aiogram.types import CallbackQuery
+
+from app.interface_student import StudentInterface
+from config import API_TOKEN
```

Comment:
Probably config should be moved to /app folder to have imports working

Reply:
fixed